estimation/estimate_wct.py

```diff
@@ -25,6 +25,7 @@
 PHYSICAL_NCORES_PER_NODE = config.qconfig[config.ConfigKeys.cores_per_node.name]
 
 CH_SAFETY_FACTOR = 1.5
+ERROR_MSG_SHAPE_MISMATCH = "Invalid input data, has to be {required_column_count} columns. One for each feature."
```

Comment:
Should we just make a custom exception with the same message, and you input the column count instead?
e.g.
class EstimationInputException(Exception):
    ERROR_MSG_SHAPE_MISMATCH = "..."
    def __init__(self, column_count, *args, **kwargs):
        super().__init__(ERRO_MSG_SHAPE_MISMATCH.format(required_column_count=column_count), *args, **kwargs)

Reply:
Ugh, thanks for nuking the formatting github

Reply:
that sounds like an overkill, at least for now. 
we have only this one kind of error at the moment, constructing our own handler seems not necessary.

Reply:
if we have a lot of checking/error handling that might be more useful.